MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontSize = 18;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with font size](images/Button_fontsize.png)
+
+### FontAttributes
+
+The `FontAttributes` property is used to customize the font style of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" FontAttributes = "Italic">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontAttributes = FontAttributes.Italic;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with fontattributes](images/Button_fontattributes.png)
+
+### FontFamily
+
+The `FontFamily` property is used to customize the font family of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" FontFamily = "Samantha-Demo">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.FontFamily = "Samantha-Demo";
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with fontfamily](images/Button_fontfamily.jpg)
+
+### TextAlignment
+
+The `HorizontalTextAlignment ` and `VerticalTextAlignment ` properties are used to customize the alignment of text in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" HorizontalTextAlignment="Center" VerticalTextAlignment="Center">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.HorizontalTextAlignment = TextAlignment.Center;
+button.VerticalTextAlignment = TextAlignment.Center;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Background Customization
+
+The background of the button can be customized by its background color, border color, border width and corner radius.
+
+### BackgroundColor
+
+The `Background` property is used to customize the background color of `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" Background = "DeepSkyBlue">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.Background = Colors.DeepSkyBlue;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with background color](images/Button_backgroundcolor.png)
+
+### Stroke
+
+The `BorderColor` property is used to customize the color of border in `SfButton`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<button:SfButton x:Name="button" Text="Button" Stroke="Red">
+</button:SfButton>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfButton button = new SfButton();
+button.Text = "Button";
+button.Stroke = Color.Red;
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with stroke](images/Button_border.png)
+
+### StrokeThickness
+
+The `BorderWidth` property is used to customize the thickness of border in `SfButton`. 
```

Comment:
The `BorderWidth` property is used to customize the thickness of the border in `SfButton`.